src/cli/commands/create.js

```diff
@@ -51,7 +52,10 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
       throw new MessageError(reporter.lang('createInvalidBin', packageName));
     }
 
-    await child.spawn(binPath, rest, {stdio: `inherit`});
+    const linkedBinPath = config.getTemp(`create-bin-${packageName}`);
+    await linkBin(binPath, linkedBinPath);
```

Comment:
I assumed we were previously not using the already linked bin for a reason. If there are no reasons, I can indeed change to use the existing one in global.